### PR TITLE
Add BPX and devtools

### PR DESCRIPTION
**Pull request by Saransh-cpp**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Saransh-cpp** [comment]
@conda-forge/python, ready for review!

File: recipes/bpx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{% set name = "bpx" %}
+{% set version = "0.4.0" %}
+{% set python_min = "3.9" %}
```
Comment (ocefpaf) [inline recipes/bpx/meta.yaml:3]: W no longer need this.

```suggestion
```

File: recipes/devtools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{% set name = "devtools" %}
+{% set version = "0.12.2" %}
+{% set python_min = "3.7" %}
```
Comment (ocefpaf) [inline recipes/devtools/meta.yaml:3]:
```suggestion
```

**Comment by Saransh-cpp** [comment]
Thanks for the review, @ocefpaf! Should be ready again!

**Comment by Saransh-cpp** [comment]
The tests are failing with -
```
conda_build.exceptions.CondaBuildUserError: Failed to render jinja template in /Users/runner/work/1/s/recipes/devtools/meta.yaml:
'python_min' is undefined
```

because there are multiple packages in this PR. Closing this and creating individual PRs such that the tests pass.

**Pull request closed without merging**